README.md

```diff
@@ -3,14 +3,14 @@
 # Mapfish Print Manager
 
 An interface manager to easily communicate with the MapFish Print module via
-an ol3 based webapplication.
+an ol based webapplication.
```

Comment:
maybe a short note about ol version compatibility could be useful here (e.g. `v1.0.0` -> `ol4`, `v2.0.0` -> `ol5`)